modules/quickstart/partials/bash/docker_run_existing_cstar.sh

```diff
@@ -0,0 +1,48 @@
+=== Starting Stargate with existing Cassandra cluster
+
+Stargate can be added to an existing Cassandra cluster operating in a
+containerized environment with the following command:
+
+[source,bash,subs="attributes+"]
+----
+docker run --name stargate \
+  -p 8080:8080 \
+  -p 8081:8081 \
+  -p 8082:8082 \
+  -p 127.0.0.1:9042:9042 \
+  -d \
+  -e CLUSTER_NAME=stargate \
+  -e SEED=127.0.0.1 \
+  -e CLUSTER_VERSION=3.11 \
+  -e SIMPLE_SNITCH=true \
+  -e ENABLE_AUTH=true \
+  stargateio/stargate-3_11:{stargate-docker-tag}
+----
+
+where
```

Comment:
Related to https://github.com/stargate/docs/issues/32#issuecomment-718781008 should we call out what's happening with `LISTEN_ADDRESS`? That if it's not provided then the start script will find it

Reply:
Incorporated.